src/sentry/migrations/0053_remove_old_functions.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-03-11 15:29
+from __future__ import unicode_literals
+
+import six
+import re
+
+from django.db import migrations
+from django.db.models import Q
+
+from sentry.utils.query import RangeQuerySetWrapper
+from sentry.api.event_search import get_function_alias
+
+FIELDS_TO_CHANGE = set(["orderby", "fields", "yAxis", "query"])
+FUNCTION_CHANGE = {
+    "p75": "p75()",
+    "p95": "p95()",
+    "p99": "p99()",
+    "apdex": "apdex(300)",
+    "impact": "impact(300)",
+    "last_seen": "last_seen()",
+    "latest_event": "latest_event()",
+}
+COUNT_REGEX = re.compile(".*(count\([a-zA-Z\._]+\)).*")
+
+
+def convert_function(field, count_default="count()", transform=None):
+    if transform is None:
+        transform = lambda x: x
+
+    if "count" in field and "count_unique" not in field:
+        field = count_default
+        return field
+
+    for old_fn, new_fn in six.iteritems(FUNCTION_CHANGE):
+        if old_fn + "()" in field:
+            field = field.replace(old_fn + "()", transform(new_fn))
+        elif old_fn in field:
+            field = field.replace(old_fn, transform(new_fn))
+
+    return field
+
+
+def convert(DiscoverSavedQuery, saved_query):
+    old_query = saved_query.query
+    new_query = {}
+
+    for key in old_query:
+        if key in FIELDS_TO_CHANGE:
+            continue
```

Comment:
Could simplify to `new_query = {key: val for (key, val) in old_query.items() if key in FIELDS_TO_CHANGE}`.

Why do we do this copy? It looks like all these values get overwritten later?

Reply:
I only overwrite a select few of the fields, not all of them. So this is to make sure I preserve all the fields that I'm not changing.